src/supercell.jl

```diff
@@ -115,7 +119,7 @@ function cell_to_supercell(scfres::NamedTuple)
     energies_supercell = [n_unit_cells * v for v in values(scfres_unfold.energies)]
```

Comment:
I would make the ham up here using `energies_hamiltonian`. This is called by `Hamiltonian( )` anyway.